src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Figure[] figures = new Figure[]{new Circle(), new IsoscelesTrapezoid(),
+            new RightTriangle(), new Rectangle(), new Square()};
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        return figures[random.nextInt(figures.length - 1)];
+    }
```

Comment:
lets randomize number in this method
and create a new Square if the number is 0, a new Circle if its 1 and so on

you should pass new random values in each constructor, to create random figures with random fields (color as well)

better make private methods to get each random figure - this would make your code easier to read